next.config.js

```diff
@@ -6,29 +8,13 @@ module.exports = {
   images: {
     remotePatterns: [
       {
-        hostname: process.env.NEXT_IMAGE_DOMAIN,
+        protocol: drupalUrl.protocol.replace(':', ''),
+        hostname: drupalUrl.hostname,
       },
     ],
   },
   typescript: {
-    ignoreBuildErrors: true
-  },
-  async rewrites() {
-    return {
-      beforeFiles: [
-        {
-          source: '/_next/image',
-          destination: '/_next/image?url=/no-image.png',
-          has: [{type: 'query', key: 'url', value: (`htt[p|ps]://${process.env.NEXT_IMAGE_DOMAIN}.*`)}],
-          missing: [{type: 'query', key: 'url', value: '(.*itok=([\\w|-]+))'}]
-        },
-        {
-          source: '/_next/image',
-          destination: '/_next/image?url=:url',
-          has: [{type: 'query', key: 'url', value: '(?<url>.*[jpg|png|jpeg|gif]\?itok=([\\w|-]+)).*'}]
-        },
-      ]
-    };
+    ignoreBuildErrors: process.env.CI !== 'true',
```

Comment:
What build errors are causing you issues?

Reply:
here it actually doesn't have an issue. but if we ever do a `yarn install --prod` in vercel, it will cause build errors. This keeps the build errors in Github Actions